index.html

```diff
@@ -1,16 +1,113 @@
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="stylesheet" href="style.css" />
     <title>Responsive Cake webpage</title>
+    <script
+      src="https://kit.fontawesome.com/02e465af65.js"
+      crossorigin="anonymous"
+    ></script>
     <!-- Add a link to your css file here -->
-</head>
+  </head>
+
+  <body>
+    <header class="header">
+      <div class="header-container">
+        <a class="header-img"><img src="/img/logo.jpg" /></a>
+        <div class="icon menu-btn">
+          <i class="fas fa-bars"></i>
+        </div>
+      </div>
+      <p class="header-desc">The best cakes in town delivered to your door.</p>
+    </header>
+
+    <nav class="navbar">
+      <ul class="navbar-list">
+        <li>
+          <a href="#">Home</a>
+        </li>
+        <li>
+          <a href="#">Cakes</a>
+        </li>
+        <li>
+          <a href="#">OrderIn</a>
+        </li>
+        <li>
+          <a href="#">Lessons</a>
+        </li>
+        <li>
+          <a href="#">About</a>
+        </li>
+      </ul>
+    </nav>
+
+    <main>
+      <!-- Main first section -->
+      <section class="hero">
+        <div class="hero-desc">
+          <h1>Welcome</h1>
+          <p>
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Tenetur sunt dignissimos illo minus, accusantium velit perspiciatis illum aspernatur cumque sequi libero modi in dolores at asperiores? Labore cumque perferendis dignissimos, mollitia enim quia optio rem corrupti sint quasi eos tenetur excepturi consequatur consectetur laudantium vel earum. Ratione sapiente saepe illo?
+          </p>
+        </div>
+        <a><img src="img/background-imagejpg.jpg" alt="background-image" /></a>
+      </section>
+      <!-- Main second section -->
+      <section class="cakes-pictures-container">
+        <a><img class="cakes-pictures" src="img/cake-1.jpg" alt="" /></a>
+        <a><img class="cakes-pictures"  src="img/cake-2.jpg" alt="" /></a>
+        <a><img class="cakes-pictures"  src="img/cake-3.jpg" alt="" /></a>
+        <a><img class="cakes-pictures"  src="img/cake-4.jpg" alt="" /></a>
+      </section>
+    </main>
 
-<body>
-    <!-- Add your markup here -->
-</body>
+    <footer>
+      <div class="footer-list">
+        <!-- Footer Basics -->
+        <div class="footer-list-basics">
+          <p>Basics</p>
+          <ul>
+            <li>
+              <a>FAQs</a>
+            </li>
+            <li>
+              <a>Delivery Partners</a>
+            </li>
+          </ul>
+        </div>
+        <!-- Footer Get In Touch -->
+        <div class="footer-list-GetInTouch">
+          <p>Get In Touch</p>
+          <ul>
+            <li>
+              <a>Contact Us</a>
+            </li>
+            <li>
+              <a>Careers</a>
+            </li>
+          </ul>
+        </div>
+        <!-- Footer Policies -->
+        <div class="class="footer-list-policies">
```

Comment:
`class="footer-list-policies"` ?

Reply:
typed class twice by mistake.

Amended the class name properly. 

`<div class="footer-list-policies">`